Converters/Revit/Speckle.Converters.RevitShared/Helpers/LevelExtractor.cs

```diff
@@ -0,0 +1,29 @@
+namespace Speckle.Converters.RevitShared.Helpers;
+
+public sealed class LevelExtractor
+{
+  // stores the map of level id to level name
+  private readonly Dictionary<DB.ElementId, string> _levelCache = new();
+
+  public LevelExtractor() { }
+
+  public string? GetLevel(DB.Element element)
```

Comment:
@bjoernsteinhagen once we merge this PR in, we should reconsider this class.

Reply:
Yes sir, thanks for the ping!